.github/workflows/package_updates.yml

```diff
@@ -161,3 +161,12 @@ jobs:
           else
             ./scripts/bin/update-packages "@all"
           fi
+      - name: Trigger repology metadata generation
+        if: always()
```

Comment:
Why even have a condition here?
This step *always* runs if there isn't a condition specified.

Reply:
> This step _always_ runs if there isn't a condition specified.

Nope. It will not run if previous step fails. This condition makes sure it runs in any case.

Like `Generate build artifacts` step [here](https://github.com/termux/termux-packages/actions/runs/14530707856/job/40769870242#step:8:29).

Reply:
Mmmh yep didn't consider that this is a sub-step.